diff --git a/very_good_dart_package/.github/workflows/main.yaml b/very_good_dart_package/.github/workflows/main.yaml
index 1259f84..857364d 100644
--- a/very_good_dart_package/.github/workflows/main.yaml
+++ b/very_good_dart_package/.github/workflows/main.yaml
@@ -15,6 +15,7 @@ jobs:
 
   build:
     uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/dart_package.yml@v1
-
+{{#publishable}}
   pana:
     uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/pana.yml@v1
+{{/publishable}}
diff --git a/very_good_dart_package/CHANGELOG.md "b/very_good_dart_package/{{#publishable}}CHANGELOG.md{{\\publishable}}"
similarity index 100%
rename from very_good_dart_package/CHANGELOG.md
rename to "very_good_dart_package/{{#publishable}}CHANGELOG.md{{\\publishable}}"
diff --git a/very_good_dart_package/LICENSE "b/very_good_dart_package/{{#publishable}}LICENSE{{\\publishable}}"
similarity index 100%
rename from very_good_dart_package/LICENSE
rename to "very_good_dart_package/{{#publishable}}LICENSE{{\\publishable}}"
